python/py-html2text/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           select 1.0
 
 name                py-html2text
 version             2020.1.16
-revision            0
+revision            1
```

Comment:
you don't / shouldn't increase the `revision` when adding a subport

Reply:
ciao reneee and thank for the comment.
i have just a doubt and I would be grateful if you confirm it.
i was under the impression that if you add/remove lang version you are supposed to increase the revision. i am doing the same for the migration of p5-* portfile from p5.* to p5.34.
if you confirm I don't have for python or whatever other reason I will be happy to do it! 
cheers!

Reply:
thank reneeotten, now it's clear! i amended the pr and resolved this conversation. cheers,
--
ferdy